server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileExporters.java

```diff
@@ -124,10 +129,15 @@ private RulesProfile wrap(QualityProfileDto profile) {
     DbSession dbSession = dbClient.openSession(false);
     RulesProfile target = new RulesProfile(profile.getName(), profile.getLanguage());
     try {
-      for (ActiveRuleDto activeRule : dbClient.activeRuleDao().selectByProfileKey(dbSession, profile.getKey())) {
+      List<ActiveRuleDto> activeRuleDtos = dbClient.activeRuleDao().selectByProfileKey(dbSession, profile.getKey());
+      List<ActiveRuleParamDto> activeRuleParamDtos = dbClient.activeRuleDao().selectParamsByActiveRuleIds(dbSession, Lists.transform(activeRuleDtos, ActiveRuleDtoToId.INSTANCE));
```

Comment:
I think we should create `ActiveRuleDtoFunctions` similar to `RuleDtoFunctions` and `ComponentDtoFunctions` with a method `toId()`.